Sources/AccessibilitySnapshot/SnapshotTesting/SnapshotTesting+Accessibility.swift

```diff
@@ -41,13 +41,15 @@ extension Snapshotting where Value == UIView, Format == UIImage {
     /// read certain views. Defaults to `true`.
     /// - parameter drawHierarchyInKeyWindow: Whether or not to draw the view hierachy in the key window, rather than
     /// rendering the view's layer. This enables the rendering of `UIAppearance` and `UIVisualEffect`s.
+    /// - parameter precision: The percentage of pixels that must match.
     /// - parameter markerColors: The array of colors which will be chosen from when creating the overlays.
     /// - parameter showUserInputLabels: Controls when to show elements' accessibility user input labels (used by Voice
     /// Control).
     public static func accessibilityImage(
         showActivationPoints activationPointDisplayMode: ActivationPointDisplayMode = .whenOverridden,
         useMonochromeSnapshot: Bool = true,
         drawHierarchyInKeyWindow: Bool = false,
+        precision: Float = 1.0,
```

Comment:
Ideally, we'd also move to a version that allows for the use of `perceptualPrecision`, as made available in 2022 [here](https://github.com/pointfreeco/swift-snapshot-testing/pull/628).

If there's any appetite for that migration, I'm happy to give it a go as well.